lib/db_helper/tasks.rb

```diff
@@ -110,22 +132,28 @@
   desc "Swap database config files"
   task :swap_config do
     databases = []
-    config_dir = Dir.new(File.join(Rails.root,'config'))
+    config_dir = Dir.new(File.join(Rails.root, 'config'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.